downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -591,7 +591,7 @@ websubhub:ContentDistributionMessage msg = {content: "This is sample content del
 var publishResponse = hubClientEP->notifyContentDistribution(msg);
 ```
 
-- Introduced an websubhub-listener configuration to configure a websubhub listener. 
+- Introduced an `websubhub-listener` configuration to configure a websubhub listener.
```

Comment:
This is not a keyword or name of a type, right? I don't think we need to emphasize this.

```suggestion
- Introduced a websubhub-listener configuration to configure a websubhub listener.
```

Reply:
Fixed